frontend/package.json

```diff
@@ -37,5 +37,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "https://dry-wildwood-77221.herokuapp.com"
+  "proxy": "http://localhost:8080"
```

Comment:
Głupio, że nie można sobie gdzieś indziej zdefiniować tego proxy, kiedyś poszukamy jakiegoś pomysłu. Bo też mi się kilka razy zdarzyło to zacommitować xD